vehicle/mkv/software/brakelight_bspd/bspd.c

```diff
@@ -0,0 +1,65 @@
+#include "libs/adc/api.h"
+#include "libs/can/api.h"
+#include "libs/timer/api.h"
+#include <avr/interrupt.h>
```

Comment:
Hmm do we still need an avr reference here? Feels like this was kinda the point of the lib rewrite?
Actually looking at: https://www.nongnu.org/avr-libc/user-manual/group__avr__interrupts.html, looks sei, cei come from here. 
I wonder if in the future we want to move these to the interrupt library as to remove the need for avr specific imports in app code

Reply:
Yeah I've thought about this a bit... I think I'm going to keep it as is for now, and we can revisit down the line if we want.